Description: Fix privacy violations
 Remove links to Flattr button.
 .
 Remove links to jQuery on Google servers and use jQuery from Debian
Author: Sunil Mohan Adapa <sunil@medhas.org>
Forwarded: not-needed
Last-Update: 2016-12-03

--- jquery-knob-1.2.13.orig/index.html
+++ jquery-knob-1.2.13/index.html
@@ -2,9 +2,9 @@
 <html>
     <head>
         <title>jQuery Knob demo</title>
-        <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.9.0/jquery.min.js"></script>
-        <!--[if IE]><script type="text/javascript" src="excanvas.js"></script><![endif]-->
-        <script src="dist/jquery.knob.min.js"></script>
+        <script src="/javascript/jquery/jquery.min.js"></script>
+        <!--[if IE]><script type="text/javascript" src="/javascript/excanvas/excanvas.min.js"></script><![endif]-->
+        <script src="/javascript/jquery-knob/jquery.knob.min.js"></script>
         <script>
             $(function($) {
 
@@ -107,7 +107,7 @@
             <h1>jQuery Knob</h1>
         </div>
         <div>
-            <p>Nice, downward compatible, touchable, jQuery dial. <a href="http://flattr.com/thing/674900/jQuery-Knob" target="_blank"><img src="http://api.flattr.com/button/flattr-badge-large.png" alt="Flattr this" title="Flattr this" border="0" /></a></p>
+            <p>Nice, downward compatible, touchable, jQuery dial.</p>
             <p style="font-size: 20px">* implemented interactions : mouse click and wheel mouse, keyboard (on focus) and fingers (touch events)</p>
         </div>
         <div class="demo">
